test/ui/login-test.js

```diff
@@ -0,0 +1,79 @@
+'use strict';
+
+/* Basic POC test to verify operation of installer login screen
+   when invalid username/password is entered
+   ldimaggi, March 2016
+   */
+
+let webdriver = browser.driver;
+let ACCOUNT_URL = '/account';
+let PAGE_TITLE = 'Red Hat Developer Platform Installer';
+let BAD_USERNAME = "badusername1";
+let BAD_PASSWORD = "badpassword1";
+var loginButton;
+
+describe('Login', function() {
+
+  beforeEach(function() {
+    loginButton = protractor.until.elementLocated(By.className('btn-lg'))
+  });

```

Comment:
This would be better suited as a beforeAll function - you only need to find the element once.